### PR TITLE
feat: group evm multichain sections

### PR DESCRIPTION
**Pull request by lukicenturi**
image

### CODE REVIEW

File: frontend/app/src/pages/settings/api-keys/external/index.vue [hunk 1]
```diff
@@ -130,33 +153,43 @@ onMounted(async () => {
...
+        <v-tab v-for="tab in evmEtherscanTabs" :key="tab.chain">
+          <evm-chain-icon :chain="tab.chain" />
+          <div class="ml-2">{{ tab.chain }}</div>
+        </v-tab>
+      </v-tabs>
+      <v-divider />
+      <v-tabs-items v-model="evmEtherscanTabIndex" class="pt-4">
+        <v-tab-item v-for="tab in evmEtherscanTabs" :key="tab.chain">
+          <service-key
+            v-model="tab.ref.value"
```
Comment (kelsos) [inline frontend/app/src/pages/settings/api-keys/external/index.vue:174]: Can we find an alternative approach here?
In general, if we have to manually unref a ref in a template then we are doing something wrong.

**Comment by kelsos** [comment]
@lukicenturi we also have to do something here about the vertical jumping when changing tabs.

**Comment by lukicenturi** [comment]
> @lukicenturi we also have to do something here about the vertical jumping when changing tabs.

should be updated now

**Review by kelsos — APPROVED** [review]
Great @lukicenturi thanks

**Pull request merged**